kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -429,9 +430,19 @@
             createPortal(
               e(MobileKeypad, {
                 style: keypadStyle.keypadContainer,
-                onElementMounted: setKeypadElement,
+                onElementMounted: el => {
+                  // We need to add the class to the container element
+                  // but the MobileKeypad component does not pass through
+                  // React's className prop to the root element.
+                  const domNode = el.getDOMNode();
+                  if (domNode) {
+                    domNode.classList.add('perseus-keypad-container');
+                  }
+                  setKeypadElement(el);
+                },
                 onDismiss: () => renderer && renderer.blur(),
                 onAnalyticsEvent: async () => {},
+                onClick: () => {},
```

Comment:
Is adding `onClick` specifically here allowing the click events to propagate properly?

Reply:
Hrm, that might be unnecessary - it was definitely one of the things I tried. Will double check.

Reply:
Yes, it is unnecessary, removing.

Reply:
Updated.